### ISSUE TITLE
As a user the footer shouldn't jump when the page loads

### ISSUE THREAD

**Issue by thescientist13** [ContributaryCommunity/www.contributary.community#17]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
When the app first loads, it makes a request for the topolgy.  When topolgy data comes back, the dropdown is populated and shows.  However, this causes the footer to jump down suddenly, which makes for a janky UX.

## Details
Setting a minimum height to the app container should not make it jump so much when data is finally loaded.

**Comment by penguin999** [comment]
@thescientist13 
Are we saying that the footer should start out where it now winds up once the Step 3 dropdown is shown?

**Comment by thescientist13** [comment]
Yeah, essentially just setting a min-height to the "body" / content of the app.  (so probably just some CSS `margin` or `padding`)
